supportedBackends/htcondor/src/main/scala/cromwell/backend/impl/htcondor/HtCondorJobExecutionActor.scala

```diff
@@ -299,10 +299,25 @@ class HtCondorJobExecutionActor(override val jobDescriptor: BackendJobDescriptor
   }
 
   private def resolveJobCommand(localizedInputs: CallInputs): Try[String] = {
-    if (runtimeAttributes.dockerImage.isDefined)
+    val command = if (runtimeAttributes.dockerImage.isDefined) {
       modifyCommandForDocker(call.task.instantiateCommand(localizedInputs, callEngineFunction, identity), localizedInputs)
-    else
+    } else {
       call.task.instantiateCommand(localizedInputs, callEngineFunction, identity)
+    }
+    command match {
+      case Success(cmd) => tellMetadata(Map("command" -> cmd))
+      case Failure(ex) =>
+        log.error("{} failed to resolve command due to exception:{}", tag, ex)
```

Comment:
You could also send this to metadata:
tellMetadata(Map(s"${CallMetadataKeys.Failures}[$randomNumberString]" -> ex.getMessage))